src/typography/Text/Text.tsx

```diff
@@ -42,12 +59,32 @@ const colorMap: Record<TextColor, string> = {
     interactive: 'tw-text-text-interactive',
 };
 
+/** @description optional background box color used when accessibility contrast is needed */
+const boxColorMap: Record<BoxColor, string> = {
+    default: '',
+    base: 'tw-bg-base',
```

Comment:
@ryancarville We don't have a box color `base` or even `base-alt`, for those background color, we already have the TextColor mapping which is right.  Also the tailwind class is wrong. We are not setting the background color on the Text component but the right Text color depending if it's using a Box token as container.

So it should be:
`neutral: 'tw-text-box-neutral-inverse'`

Reply:
@imoutaharik understood.  Will make the required changes.